build.sbt

```diff
@@ -123,9 +123,7 @@ lazy val core: Project = Project(
   moduleName := "featran-core",
   description := "Feature Transformers",
   libraryDependencies ++= Seq(
-    "com.twitter" %% "algebird-core" % algebirdVersion excludeAll ("org.typelevel" %% "algebra"),
-    "org.scalanlp" %% "breeze" % breezeVersion,
```

Comment:
Are we getting rid of `breeze` intentionally?

Reply:
ah no ... just noticed that. i'll fix it

Reply:
done